Web/HTML Assignment-1/Question-1.html

```diff
@@ -0,0 +1,193 @@
+<!DocType html>
+<html>
+<head>
+<title>New Account</title>
+</head>
+<body>
+	<form action="#">
+		<table align="center">
+			<tr>
+				<td colspan="2"><b>Create Your Account:</b></td>
+			</tr>
+			<tr>
+				<td align="right">Email ID:</td>
+				<td><input type="email" name="email"
+					placeholder="Enter your email"></td>
+			</tr>
+			<tr>
+				<td align="right">Password</td>
+				<td><input type="password" name="password"></td>
+			</tr>
+			<tr>
+				<td align="right">Confirm Password</td>
+				<td><input type="password" name="confirm-password"></td>
```

Comment:
name should be as per naming conventions that we should in any programming language.